cmd/tracee-ebpf/main.go

```diff
@@ -359,18 +349,21 @@ func prepareBpfObject(config *tracee.Config, kConfig *helpers.KernelConfig, OSIn
 
 	// (5) no BPF file given & no BTF available & no embedded BTF: non CO-RE BPF
 
+	tVersion = strings.ReplaceAll(version, "\"", "")
+	tVersion = strings.ReplaceAll(tVersion, ".", "_")
+	kVersion = OSInfo.GetOSReleaseFieldValue(helpers.OS_KERNEL_RELEASE)
+	kVersion = strings.ReplaceAll(kVersion, ".", "_")
+
+	bpfFilePath = fmt.Sprintf("/tmp/tracee/tracee.bpf.%s.%s.o", kVersion, tVersion)
```

Comment:
I would switch this hardcoded /tmp/tracee to `traceeInstallPath`

Reply:
I'll fix things you've noticed into a new PR.